File: src/Core/ServiXpress.Domain/Servicio.cs
using ServiXpress.Domain.Common;
using System.ComponentModel.DataAnnotations.Schema;


namespace ServiXpress.Domain
{
    public class Servicio 
    {
        public int Id { get; set; }
        public string Estado { get; set; }
        public string Municipio { get; set; }
        public string Telefonos { get; set; }
        public string Correos { get; set; }
        public string OtrosMediosContacto { get; set; }
        public string Descripcion { get; set; }
        public string UbicacionMaps { get; set; }
        public float Precio { get; set; }

        public DateTime FechaHoraRegistro { get; set; }
        public DateTime? FechaVencimiento { get; set; }

        [ForeignKey("Usuario")]
        public string UsuarioId { get; set; }
        public Usuario Usuario { get; set; }

        [ForeignKey("CategoriaServicio")]
        public int CategoriaId { get; set; }
        public CategoriaServicio CategoriaServicio { get; set; }

        [ForeignKey("TipoServicio")]
        public string Tipo { get; set; }
        public TipoServicio TipoServicio { get; set; }
    }
}

File: src/APPMovil/src/Interfaces/Servicio.ts
import { Usuario } from "./Usuario";

export interface Servicio {
    id: number;
    estado: string;
    municipio: string;
    telefonos: string;
    correos: string;
    otrosMediosContacto: string;
    descripcion: string;
    ubicacionMaps: string;
    precio: number;
    fechaHoraRegistro: Date;
    fechaVencimiento?: Date | null;
    usuarioId: string;
    usuario: Usuario;
    categoriaId: number;
    categoriaServicio: CategoriaServicio;
    tipo: string;
    tipoServicio: string;
}


export interface CategoriaServicio {
    id: number
    nombre: string
    fechaHoraRegistro?: Date
}

export interface TipoServicio {
    tipo: string
}

export interface ServiceCreate {
    estado: string;
    municipio: string;
    telefonos: string;
    correos: string;
    otrosMediosContacto: string;
    descripcion: string;
    precio: number;
    usuarioId: string;
    categoriaId: number;
    tipo: string;
}